lib/python/qmk/cli/new/keyboard.py

```diff
@@ -66,18 +66,19 @@ def new_keyboard(cli):
     keyboard_type = cli.args.type if cli.args.type else choice('Keyboard Type:', KEYBOARD_TYPES, default=0)
 
     # Get username
-    user_name = None
-    while not user_name:
-        user_name = question('Your GitHub User Name:', default=find_user_name())
+    user_name = cli.args.username
+    if user_name:
+        cli.log.info(f'Using GitHub User Name: {user_name}')
+    else:
+        user_name = question('Your GitHub User Name:', default=cli.config.user.name if cli.config.user.name else git_get_username())
 
         if not user_name:
             cli.log.error('You didn\'t provide a username, and we couldn\'t find one set in your QMK or Git configs. Please try again.')
+            return False
 
-            # Exit if passed by arg
-            if cli.args.username:
-                return False
-
-    real_name = None
+    real_name = cli.args.realname
+    if real_name:
+        cli.log.info(f'Using real name: {real_name}')
     while not real_name:
```

Comment:
```suggestion
    else:
```

Reply:
Yes, we could do that too. But that is not directly related to the bug fixed by this changeset. It could be a separate changeset.